build.gradle.kts

```diff
@@ -0,0 +1,71 @@
+import net.minecraftforge.gradle.user.tweakers.ClientTweaker
+import net.minecraftforge.gradle.user.tweakers.TweakerExtension
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+import org.spongepowered.asm.gradle.plugins.MixinGradlePlugin
+
+buildscript {
+    repositories {
+        maven { setUrl("http://files.minecraftforge.net/maven") }
+        maven { setUrl("http://repo.spongepowered.org/maven") }
+    }
+
+    dependencies {
+        classpath("net.minecraftforge.gradle:ForgeGradle:${Versions.forge_gradle}")
+        classpath("org.spongepowered:mixingradle:${Versions.mixin_gradle}")
+    }
+}
+
+group = "cc.hyperium"
+version = "1.0-SNAPSHOT"
+
+plugins {
+    kotlin("jvm") version Versions.kotlin
+}
+
+apply {
+    plugin<ClientTweaker>()
+    plugin<MixinGradlePlugin>()
+}
+
+tasks.withType<KotlinCompile> {
+    kotlinOptions.jvmTarget = "1.8"
+}
+
+java {
+    sourceCompatibility = JavaVersion.VERSION_1_8
+    targetCompatibility = JavaVersion.VERSION_1_8
+}
+
+configure<TweakerExtension> {
+    setSuppressVersionTest(true)
+
+    version = "1.8.9"
+    mappings = "stable_22"
+
+    runDir = rootProject.file("minecraft").path
```

Comment:
Why are you changing the run directory of the project? I believe it was fine as `run/`, and that is the default, so most people will be looking for it there.

Reply:
The `run/` directory is just the default. I changed it to `minecraft/` because of personal preferences, and to make it very obvious what the folder is, as `run` is a bit ambiguous.

Reply:
I'd like to get input on this, what is preferred from our developers?